includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -1395,6 +1395,11 @@ public function cancel_authorization( $order ) {
 	 * @return array          The level 3 data to send to Stripe.
 	 */
 	public function get_level3_data_from_order( $order ) {
```

Comment:
I'd suggest feeding merchant country a parameter to better clarify that data generation depends on merchant county.
Something like `public function get_level3_data_from_order( string $merchant_country, WC_Order$order ): array`.

Reply:
Good suggestion.

Note: The implication of type hinting the `$merchant_country` parameter is that I need to make sure `get_account_country()` of the mocked `WC_Payment_Gateway_WCPay::$account` returns non-null value; hence, the changes in the unit test you see in the diff.